seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -460,6 +612,7 @@
         "You can set an element's font by using the <code>font-family</code> property.",
         "For example, if you wanted to set your <code>h2</code> element's font to <code>Sans-serif</code>, you would use the following CSS:",
```

Comment:
Per a later challenge's instructions, this should be `Sans-Serif`. I know it's case-insensitive, but for consistency.